service/controller/resource/updatemachinedeployments/create.go

```diff
@@ -40,20 +40,6 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 
 		var updated bool
 
-		// Syncing the Provider Operator version. For instance aws-operator,
-		// kvm-operator or the like.
```

Comment:
This does not make sense since the provider specific operator version label is not part of the CAPI CRs anymore.